docs/getting-started.md

```diff
@@ -4,32 +4,63 @@ title: Getting Started
 sidebar_label: Getting Started
 ---
 
-Celerity allows you to write highly parallel applications that can be run on
-a cluster of accelerator nodes. It focuses on providing a way of scaling
-applications to many nodes without having to be an expert in cluster
-programming. In fact, the Celerity API does not make it apparent that a program is
-(potentially) running on many nodes at all: There is no notion of _ranks_;
-partitioning of work and data is taken care of transparently behind the scenes.
-This lets you focus on your actual work, without having to concern yourself with the
-complexities of modern distributed memory cluster programming.
-
-While ease of use is one of Celerity's main goals, simplicity can only go so
-far without sacrificing considerable performance. Proficiency in modern C++
-as well as at least a rough understanding of how accelerator (GPU) programming
-differs to parallel CPU programming is required to make efficient use of Celerity.
-Lastly, you will require a good understanding of the algorithms and techniques you
-intend to implement using Celerity in order for the runtime system to be able
-to run it on a cluster both correctly and in an efficient manner.
+Celerity is a high-level C++17 API and runtime environment that aims to bring
+the power and ease of use of [SYCL](https://www.khronos.org/sycl/) to
+distributed-memory accelerator clusters.
+
+> If you want to get your hands dirty right away, move on to the
+> [Installation](installation.md) guide.
+
+## Transparently Scale Your Applications
```

Comment:
I really like grouping this long block of text and breaking it up into more digestible chunks with individual headlines. Nicely done.